### ISSUE TITLE
Wire transfer payments

### ISSUE THREAD

**Issue by federicosan** [mercadopago/dx-nodejs#163]
**Is your feature request related to a problem? Please describe.**
I would like to know if these is possible with the sdk, or if the api supports it.

**Describe the solution you'd like**
Buyers can pay by making a wire transfer from their bank account to the one specified.

**Describe alternatives you've considered**
None

**Additional context**
Not necessary

**Comment by LuKks** [comment]
I don't think you will get a payment notification when you receive a bank transfer.
It's like the investments, they are separated from the summary.
You already received a bank transfer? It appears in the summary?

Maybe it can be done checking the report of movements of the account but I don't know what information we have, obviously the amount and time but what more? the sender cbu?
Depending on the information you can do a simple match to know that someone truly paid you otherwise can be insecure until MP really implement it.

[/es/guides/localization/payment-methods](https://www.mercadopago.com.ar/developers/es/guides/localization/payment-methods)
